Workspaces/Base/ActionMaps/WorkspaceInput.cs

```diff
@@ -7,9 +7,7 @@ namespace UnityEngine.InputNew
 	public class WorkspaceInput : ActionMapInput {
 		public WorkspaceInput (ActionMap actionMap) : base (actionMap) { }
 		
-		public ButtonInputControl @primaryLeft { get { return (ButtonInputControl)this[0]; } }
-		public ButtonInputControl @primaryRight { get { return (ButtonInputControl)this[1]; } }
-		public ButtonInputControl @secondaryLeft { get { return (ButtonInputControl)this[2]; } }
-		public ButtonInputControl @secondaryRight { get { return (ButtonInputControl)this[3]; } }
+		public ButtonInputControl @secondaryLeft { get { return (ButtonInputControl)this[0]; } }
```

Comment:
I would suggest renaming this to what the action is, potentially `moveResize` or something that doesn't indicate what button is actually being pressed.